src/python/pants/backend/python/lint/pylint/rules.py

```diff
@@ -267,6 +267,8 @@ async def pylint_lint(
 
     # We batch targets by their interpreter constraints to ensure, for example, that all Python 2
     # targets run together and all Python 3 targets run together.
+    # Note that Pylint uses the AST of the interpreter that runs it. So, we include any plugin
```

Comment:
```suggestion
    # Note that Pylint uses the `ast` module of the interpreter that runs it. So, we include any plugin
```

Reply:
It actually uses Astroids rather than `ast`. I do mean AST as in a generic sense.